src/blocks/plugins/image-extension/edit.js

```diff
@@ -103,20 +103,17 @@ const Edit = ({
 	const [ cssNodeName, setNodeCSS ] = useCSSNode();
 	useEffect( () => {
 		setNodeCSS([
-			`img {
+			attributes.boxShadow ? `img {
 				box-shadow: ${ attributes.boxShadowHorizontal }px ${ attributes.boxShadowVertical }px ${ attributes.boxShadowBlur }px ${ getShadowColor() }
 			}
-			` ]);
-	}, [ attributes.boxShadowHorizontal, attributes.boxShadowVertical, attributes.boxShadowBlur, attributes.boxShadowColor, attributes.boxShadowColorOpacity ]);
+			` : '' ]);
+	}, [ attributes.boxShadowHorizontal, attributes.boxShadowVertical, attributes.boxShadowBlur, attributes.boxShadowColor, attributes.boxShadowColorOpacity, attributes.boxShadow ]);
 
 
 	return (
 		<Fragment>
-			{ attributes.boxShadow ? (
-				<BlockEdit {...props } className={ props.className + ` ${cssNodeName};` } />
-			) : (
-				<BlockEdit {...props } className={ props.className + ` ${cssNodeName};` } />
-			) }
+
+			<BlockEdit {...props } className={ props.className + ` ${cssNodeName}` } />
```

Comment:
Shouldn't we just return <BlockEdit> without modifying the className if boxShadow isn't enabled?

Reply:
I see no need for it. It does not influence the frontend. And if we already created create the node, let's just use it.